### PR TITLE
Feat: Added my json data file

### PR DESCRIPTION
**Pull request by ScottDeev**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2928"></a>

### CODE REVIEW

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please change your filename to match the case of your GitHub ID.
`data/ScottDeev.json`

**Review by dan-mba — DISMISSED** [review]
Looks good to me

**Comment by ScottDeev** [comment]
> Please change your filename to match the case of your GitHub ID. `data/ScottDeev.json`

Done

File: data/ScottDeev.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhishek-gupta-95275b233",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/scottDeev" ,
```
Comment (Panquesito7) [inline data/ScottDeev.json:20]: There's an extra space here (not sure if it'll get fixed automatically).
```suggestion
      "url": "https://github.com/scottDeev",
```

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 

Great collaboration 💪 

You can also join us in the EddieHub Discord http://discord.eddiehub.org

**Comment by ScottDeev** [comment]
> Looks good 👍
> 
> Great collaboration 💪
> 
> You can also join us in the EddieHub Discord http://discord.eddiehub.org

Thanks @eddiejaoude. 

Will be joining the Discord right away!

**Pull request merged**